odl/discr/discr_ops.py

```diff
@@ -455,8 +455,10 @@ def _call(self, x, out=None):
         >>> f = par_deriv.domain.element(data)
         >>> par_div_f = par_deriv(f)
         >>> print(par_div_f)
-        [[0.0, 1.0, 2.0, 3.0, 4.0],
-         [0.0, 1.0, 2.0, 3.0, 4.0]]
+        [
```

Comment:
Can you fix this @adler-j? You probably remember the old state, I don't feel like trying around.